tests/test.js

```diff
@@ -0,0 +1,13 @@
+const supertest = require('supertest');
+const tape = require('tape');
+const app = require('../src/app.js');
+
+tape('GET /', (t) => {
+  supertest(app)
+    .get('/')
+    .expect(200)
+    .end((err, res) => {
+      t.error(err);
```

Comment:
At the moment this test gives the text `null` which isn't super helpful, could just add a message like:

` t.error(err, 'Error test');`

Also I guess this is just a initial set up, but you're not really testing anything here, just that you don't get an error. 

Could have a check to test the right response is sent back - check that the `res.text` includes something from your template?